src/main.rs

```diff
@@ -10,6 +10,7 @@ fn main() {
         dotenv::var("DISCORD_TOKEN").expect("DISCORD_TOKEN must be set"),
         dotenv::var("HEROKU_API_KEY").expect("HEROKU_API_KEY must be set"),
         dotenv::var("BUILD_CHECK_INTERVAL").expect("BUILD_CHECK_INTERVAL must be set"),
+        dotenv::var("BUILD_MESSAGE_DISPLAY_INTERVAL").expect("BUILD_CHECK_INTERVAL must be set"),
```

Comment:
The `.env.sample` needs to be updated with this variable.

Also, let's remember to also add it on ECS :)